Contributing.md

```diff
@@ -24,34 +24,30 @@ Here's what you need to do to add or edit tutorials 👇:
    - Follow the outline in the template as you create the tutorial.
    - After the tutorial is complete, add necessary information to [index.toml](/index.toml). Here, `weight` is the order in which your tutorial appears. For example, a tutorial with `weight = 15` comes after a tutorial with `weight = 10` and before `20`. Each tutorial comes with a Google Colab link and `Open in Colab` button on the top of the tutorial by default. If your new tutorial cannot be run on Google Colab, set `colab = false` not to display `Open in Colab` button on top the tutorial.
 3. If you're **editing** an existing tutorial:
-   - Make necessary changes in the `.ipynb` file of the tutorial and save them. You don't need to make changes in the markdown file. 
-4. Run `python scripts/generate_markdowns.py --index index.toml --notebooks tutorials/your-tutorial.ipynb --output markdowns/`. This generates or updates the relevant markdown file. Pre-commit hooks will ensure the `markdowns` folder reflects your changes. Don't forget to add changed files to your commit if pre-commit hooks updated `.ipynb` or `.md` files.
-5. Create a pull request.
-6. Wait for the [CI](#ci-continuous-integration) checks to pass.
-   These checks pass if the relevant markdown files are created.
-7. Update the [README](./README.md), if necessary.
-8. Wait for a review and merge 🎉. Thank you for contributing 💙.
+   - Make necessary changes in the `.ipynb` file of the tutorial and save them.
+4. Create a pull request.
+5. Wait for the [CI](#ci-continuous-integration) checks to pass.
+6. Update the [README](./README.md), if necessary.
+7. Wait for a review and merge 🎉. Thank you for contributing 💙.
 
 ## Slugs
 
-The default behaviour for markdown files is that it gets the same name as the corresponfing `.ipynb` notebook of that tutorial. The name of the markdown file is also the location at which the tutorial will appear on the website. 
-For example, "01_Basic_QA_Pipeline" will be on https://haystack.deepset.ai/tutorials/01_basic_qa_pipeline
+By default, the name of each tutorial is also the location at which the tutorial will appear on the website. For example, "01_Basic_QA_Pipeline" will be on https://haystack.deepset.ai/tutorials/01_basic_qa_pipeline.
```

Comment:
Should we say: .. the name of each tutorial becomes the slug of its page on on the website.